### PR TITLE
Adding ProposalCommentVote feature.

### PR DESCRIPTION
**Pull request by NabeelValapra**
Todo's:
    * Request to be made through AJAX.

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -150,10 +150,10 @@
...
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': os.environ.get('DB_NAME', ''),
-        'USER': os.environ.get('DB_USER', ''),
-        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
-        'HOST': os.environ.get('DB_HOST', ''),
+        'NAME': os.environ.get('DB_NAME', 'junctionIndia'),
+        'USER': os.environ.get('DB_USER', 'postgres'),
+        'PASSWORD': os.environ.get('DB_PASSWORD', 'postgres'),
+        'HOST': os.environ.get('DB_HOST', 'localhost'),
```
Comment (sivaa) [inline settings/common.py]: Not required.

File: junction/templates/proposals/detail/base.html [hunk 2]
```diff
@@ -178,4 +178,7 @@
     });
 
 </script>
+{% block comment_vote_script %}
+    
+{% endblock comment_vote_script %}
```
Comment (sivaa) [inline junction/templates/proposals/detail/base.html]: Not required. Use     {% block script_extra %} {% endblock %}

File: junction/proposals/models.py [hunk 3]
```diff
@@ -125,7 +125,18 @@ class ProposalComment(TimeAuditModel):
     deleted = models.BooleanField(default=False, verbose_name="Is Deleted?")
 
     def __unicode__(self):
-        return "[{}] {}".format(self.commenter.get_full_name(), self.proposal)
+        return self.comment
```
Comment (sivaa) [inline junction/proposals/models.py]: Revert this change

**Comment by sivaa** [comment]
Can you also fix the Travis build and attach the screenshot of new UI?

File: settings/common.py (hunk 1 above)
Comment (NabeelValapra) [inline settings/common.py]: So can I put this file in .gitignore?

Comment (sivaa) [inline settings/common.py]: Use dev.py instead.


Thanks & Regards,
Sivasubramaniam Arunachalam

File: junction/templates/proposals/detail/base.html (hunk 2 above)
Comment (NabeelValapra) [inline junction/templates/proposals/detail/base.html]: It affects the "ProposalVoting" section.

Comment (sivaa) [inline junction/templates/proposals/detail/base.html]: You can add your script at the bottom of this block. Don't override it.


Thanks & Regards,
Sivasubramaniam Arunachalam

**Comment by NabeelValapra** [comment]
Here is the new UI, Can you fix the travis build?

ui

**Comment by theskumar** [comment]
Hey @NabeelValapra, since it's your PR branch ideally you should resolve it. That way, you'll learn how to do it and be careful next time.

I see the [errors](https://travis-ci.org/pythonindia/junction/builds/46402038) are mainly because you are not following the [coding conventions](https://github.com/pythonindia/junction/blob/master/CONTRIBUTING.md#conventions).

To reproduce the error, after you have installed requirements from `requirements-dev.txt`, run the following command `flake8` in your repo. It will list down the file names and line number where you need name fixes.

Read https://www.python.org/dev/peps/pep-0008/ for detailed conventions that are followed.

Most of the issues will never happen if you have configured your text editor the correct way. We have `.editorconfig` included in this repo, read http://editorconfig.org/ know more about it, and how it can help you.

If you happen use sublime-text editor, make sure you have these sane preferences added, you can find my settings [here](https://gist.github.com/theskumar/9ae8bfc510edbcf6c7dc).

```
...
"ensure_newline_at_eof_on_save": true,
"translate_tabs_to_spaces": true,
"trim_trailing_white_space_on_save": true,
...
```
cheers,

**Comment by theskumar** [comment]
Also, make sure you add yourself to https://github.com/pythonindia/junction/blob/master/CONTRIBUTORS.txt :blush:

**Comment by sivaa** [comment]
Thanks.
